README.md

```diff
@@ -40,189 +40,113 @@ When using GitLink, the user no longer has to specify symbol servers. The only r
 
-GitLink supports the following providers out of the box (will auto-detect based on the url):
+## Command line tool
 
-* <a href="https://bitbucket.org/" target="_blank">BitBucket</a>
-* <a href="https://github.com/" target="_blank">GitHub</a>
-* Custom Provider (custom urls)
-
-Providers currently being worked on:
-
-* <a href="https://www.assembla.com/home" target="_blank">Assembla</a>
-* <a href="http://beanstalkapp.com/" target="_blank">Beanstalk</a>
-* <a href="http://www.cloudforge.com/" target="_blank">CloudForge</a>
-* <a href="https://www.codebasehq.com/" target="_blank">Codebase</a>
-* <a href="https://www.fogcreek.com/kiln/" target="_blank">FogCreek</a>
-* <a href="https://plan.io/" target="_blank">Planio</a>
-* <a href="http://projectlocker.com/" target="_blank">ProjectLocker</a> 
-* <a href="https://rhodecode.com/" target="_blank">RhodeCode</a>
-* <a href="https://unfuddle.com/" target="_blank">Unfuddle</a>   
-
-It is also possible to specify a custom url provider.
+If you want to install the tool on your (build) computer, the package is available via <a href="https://chocolatey.org/" target="_blank">Chocolatey</a>. To install, use the following command:
 
-# Using GitLink as command line tool
+    choco install gitlink
 
 Using GitLink via the command line is very simple:
 
-1. Build the solution - in release mode with pdb files enabled
-2. Run the console application with the right command line parameters
-3. Include the PDB in your nuget package
-
-See [Oren Novotony's blog post](https://oren.codes/2015/09/23/enabling-source-code-debugging-for-your-nuget-packages-with-gitlink/) for even more detail and examples on build integration.
-
-## Most simple usage
-
-This is the most simple usage available **starting from 2.2.0**. It will automatically determine the url and commit based on a local *.git* directory.
-
-	GitLink.exe c:\source\catel
-
-## Running for the default branch
-
-    GitLink.exe c:\source\catel -u https://github.com/catel/catel 
-
-This will use the default branch (which is in most cases **master**). You can find out the default branch by checking what branch is loaded by default on the GitHub page.
-
-## Running for a specific branch
-
-    GitLink.exe c:\source\catel -u https://github.com/catel/catel -b develop
-
-This will use the develop branch.
-
-## Running for a specific branch and configuration
-
-    GitLink.exe c:\source\catel -u https://github.com/catel/catel -b develop -c debug
-
-This will use the develop branch and the debug configuration.
-
-## Running for a specific solution only
-
-Sometimes a repository contains more than 1 solution file. By default, all solutions will be processed. To only process a single solution file, use the *-f* option: 
-
-	GitLink.exe c:\source\catel -u https://github.com/catel/catel -f Catel.sln
-
-## Ignoring projects and explicitly including them
+    gitlink.exe <pdbfile>
```

Comment:
We can have a usage that shows looping over pdb's and invoking this gitlink.exe repeatedly for each one. Or we could add wildcard support, I suppose. But I wonder how many people will actually use this syntax anyway, given the improved msbuild integration we have that will do it for an entire solution as it builds. Do you have any intuition in that area?